### PR TITLE
Update 07-cond.md

### PR DESCRIPTION
**Pull request by ognancy4life**
remove extra space in print statement

### CODE REVIEW

File: _episodes/07-cond.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Python simply does nothing if the test is false:
 num = 53
 print('before conditional...')
 if num > 100:
-    print(num,' is greater than 100')
+    print(num,'is greater than 100')
```
Comment (ldko) [inline _episodes/07-cond.md:66]: Good catch, Nancy! Would you mind also adding in a space between the comma and quote, for consistency with the spacing used in the lesson?

**Review by ldko — COMMENTED** [review]
Thanks for submitting a fix for this, Nancy.

File: _episodes/07-cond.md (hunk 1 above)
Comment (ognancy4life) [reply _episodes/07-cond.md]: Done! This is my first time contributing this way. Does the additional commit and this comment suffice or should I do something more? Thanks!

Comment (ldko) [reply _episodes/07-cond.md]: What you did is what I had hoped for. Thank you!

**Review by ldko — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**